src/components/Documents/PipelineConfigForm.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <q-dialog ref="dialogRef" no-esc-dismiss no-backdrop-dismiss>
+      <q-layout view="lhh LpR lff" container style="height: 90%">
+        <q-page-container>
+          <q-page style="padding-top: 50px">
+            <JsonEditorVue
+              v-model="jsonConfig"
+              v-bind="{/* local props & attrs */}"
+            />
+          </q-page>
+          <q-page-sticky expand position="top">
+            <q-toolbar class="bg-secondary text-white">
+              <q-toolbar-title>
+                {{ t('configurationOfPipeline') }}
+                <q-btn flat round dense icon="download" @click="downloadConfig">
+                  <q-tooltip :delay="500">
+                    {{ t('downloadConfig') }}
+                  </q-tooltip>
+                </q-btn>
+              </q-toolbar-title>
+              <div class="q-gutter-sm">
+                <q-btn flat round dense icon="done" @click="saveConfig" />
+                <q-btn flat round dense icon="close" @click="closeConfig"/>
+              </div>
+            </q-toolbar>
+          </q-page-sticky>
+        </q-page-container>
+      </q-layout>
+  </q-dialog>
+</template>
+
+<script setup lang="ts">
+import JsonEditorVue from 'json-editor-vue'
+import { inject, onMounted, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { ConceptPipelineApiKey } from 'boot/axios'
+import useNotify from 'src/mixins/useNotify'
+import { Notify, useDialogPluginComponent } from 'quasar'
+
+const { dialogRef, onDialogOK, onDialogCancel } = useDialogPluginComponent()
+const { renderError } = useNotify()
+const conceptPipelineApi = inject(ConceptPipelineApiKey)
+const { t } = useI18n()
+const jsonConfig = ref<object|string>('')
+const props = defineProps({
+  pipelineId: {
+    type: String,
+    default: undefined
+  },
+  savedConfig: {
+    type: String,
+    default: ''
+  },
+  language: {
+    type: String,
+    default: 'en'
+  }
+})
+const pipelineConfigError = ref(false)
+
+onMounted(() => {
+  loadConfig()
+})
+
+function loadConfig() {
+  let errStr = t('conceptCluster.configurationGetError')
+  let defaultConfig = props.pipelineId === undefined
+  pipelineConfigError.value = true
+  conceptPipelineApi?.getConceptGraphPipelineConfiguration(props.pipelineId, props.language)
+    .then((r) => {
+      //ToDo: should be ResponseEntity<String> but is parsed (automatically?) as Object/JSON?
+      if (!r.data || r.data.length === 0) {
+        jsonConfig.value = errStr
```

Comment:
Better nest it inside an object, otherwise it is quite hard to guess how to change the configuration. Also, for a string value, the arrow on the right is not visible. This also applies to line 88.

```suggestion
        jsonConfig.value = { error: errStr }
```

Reply:
Ah, I see! I thought you referred to something `js`/`quasar `related. Thanks!